src/player.js

```diff
@@ -2072,7 +2072,8 @@ export default class Player extends FakeEventTarget {
     }
     this.ready()
       .then(() => {
-        if (this.isLive() && (!this.isDvr() || (typeof this.currentTime === 'number' && this.currentTime < 0))) {
+        const liveOrDvrOutOfDvrWindow = (this.isLive() && (!this.isDvr() || (typeof this.currentTime === 'number' && this.currentTime < 0)));
+        if (!this._adsController.isAdBreak() && liveOrDvrOutOfDvrWindow) {
           this.seekToLiveEdge();
```

Comment:
should seek once the ad is done no?

Reply:
I don't think so I think DAI keeps you on the edge when you're playing ads.

Reply:
what is the user paused for more than the non-dvr window (the actuall buffer length)
this change will try to play out of buffer content

Reply:
and need to test on safari as well as it plays natively

Reply:
Fix by after ad break end I'm seeking to live edge - tested on safari as well